tests/test-user.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+require("dotenv").load();
+
+const crypto = require("crypto");
+const test = require("tape");
+
+const models = require("../db/models");
+
+test("Test building user autogenerates verification token", t => {
+  t.plan(2);
+  const user = models.User.build({ email: "test@test.com" });
+  t.notEqual(user.verificationToken, null, "verificationToken is not null");
+  t.equal(user.verificationToken.length, 80, "verificationToken is expected length");
+});
+
+test("Test building user validates email", t => {
+  t.plan(2);
+  const valid_email = "test@test.com";
+  const invalid_email = "asdfghjkl";
+
+  let user = models.User.build({ email: valid_email });
+  user.validate().then(user=>{
```

Comment:
Not sure if it's worth using `async`/`await` in the tests to be consistent with the other code. I don't feel strongly either way.